docs/aws-cloud9.md

```diff
@@ -0,0 +1,313 @@
+# Setting up Telescope in AWS Cloud9 IDE
+
+**Disclaimer**: The EC2 instance used in this guide is not within AWS's Free-Tier so please see [EC2 Pricing](https://aws.amazon.com/ec2/pricing/on-demand/) to see if you're comfortable with these costs. Cloud9 has a cost-saving setting to help reduce costs by automatically hibernating after 30 minutes of inactivity. Running Gatsby and Docker in development is CPU intensive so these are the EC2 instances I recommend:
+
+- Minimum: `t2.medium (4 GiB RAM + 2 vCPU)`
+- Recommended: `t2.large (8 GiB RAM + 2 vCPU)`
+
+**Summary of Pricing**:
+
+- t2.medium costs \$0.0464 per hour
+- t2.large costs \$0.0928 per hour
+- 30GB Amazon Elastic Block Storage (EBS) costs \$3 per month
+
+**Example Usage**:
```

Comment:
So you're showing Cost Estimate Per Month?  Can we label it as such?

Reply:
Good idea.

They're labeled now :)